server.js

```diff
@@ -127,6 +133,15 @@ async function lookupQuiz(quizID, completion) {
         completion(result[0])
     });
 }
+//description tags name
+async function searchCollection(searchTerm, completion) {
+    var query = searchTerm.split('+').join(' ');
+    database.collection(quizCollection).find({$or: [{name: {$regex: query}},{description: {$regex: query}},{tags: {$in: [query]}}]}).toArray(function(err, result) {
```

Comment:
Can you use the mongobd search functionality please? This looks very brittle to me. https://docs.mongodb.com/manual/text-search/